### ISSUE TITLE
Upgrade ktlint version to 0.40.0

### ISSUE THREAD

**Issue by dineshbhagat** [JLLeitschuh/ktlint-gradle#428]
This will address [this](https://github.com/JLLeitschuh/ktlint-gradle/issues/414) issue as well

[Ktlint 0.40.0 Release](https://github.com/pinterest/ktlint/releases/tag/0.40.0)

**Comment by emrisb** [comment]
> This will address [this](https://github.com/JLLeitschuh/ktlint-gradle/issues/414) issue as well
> 
> [Ktlint 0.40.0 Release](https://github.com/pinterest/ktlint/releases/tag/0.40.0)

We are waiting this update. Please could you upgrade?

**Comment by Tapchicoma** [comment]
@emrisb you could set version via:
```kotlin
ktlint {
    version.set("0.40.0")
}
```

Baseline support would be added separately from KtLint default version update.

**Comment by emrisb** [comment]
> @emrisb you could set version via:
> 
> ```kotlin
> ktlint {
>     version.set("0.40.0")
> }
> ```
> 
> Baseline support would be added separately from KtLint default version update.

I already set ktlint version, but --baseline does not work.
When i run **./gradlew ktlintCheck --baseline=ktlint-baseline.xml**,
it gives **Unknown command-line option '--baseline'.**

Also i tried custom ktlint task like below:
./gradlew ktlintTest --baseline=ktlint-baseline.xml
```
task ktlintTest(type: JavaExec, group: LifecycleBasePlugin.VERIFICATION_GROUP) {
    description = "Check Kotlin code style."
    classpath = configurations.ktlint
    main = 'com.pinterest.ktlint.Main'
    args '*/src/**/*.kt', '--baseline=ktlint-baseline.xml'
}
```
This time it gives following message:
> Task :ktlintTest FAILED
Exception in thread "main" picocli.CommandLine$UnmatchedArgumentException: Unknown option: --baseline=ktlint-baseline.xml
        at picocli.CommandLine$Interpreter.parse(CommandLine.java:7283)
        at picocli.CommandLine$Interpreter.parse(CommandLine.java:7147)
        at picocli.CommandLine.parseArgs(CommandLine.java:836)
        at com.pinterest.ktlint.Main.main(Main.kt:55)

**Comment by Tapchicoma** [comment]
@emrisb this issue is only about updating default KtLint version in plugin extension configuration. For baseline support, please track #414

**Comment by Tapchicoma** [comment]
Closed via #421
